lib/src/main/java/org/team100/lib/commands/drivetrain/DriveWithProfile2.java

```diff
@@ -82,14 +83,14 @@ public void initialize() {
         ySetpoint = m_swerve.getState().y();
         thetaSetpoint = m_swerve.getState().theta();
 
-        Optional<Pose2d> opt = m_fieldRelativeGoal.get();
+        Optional<Pose2d> opt = m_fieldRelativeGoalSupplier.get();
```

Comment:
it looks to me like this command only does the "coordinated profile" thing if it happens to run with a goal the first time it is "initialized".  also, note that the coordination scaling changes the profiles, so if you press the button over and over, the profiles will get slower and slower.  maybe take out all this initialization coordination stuff until the basic uncoordinated profile is working perfectly?

Reply:
I figured the whole calculation might take too long to run each cycle, however I could be wrong about that, so i'll change that, the profile does not get slower and slower due to the fact the scaling is undone in the end function.

Reply:
tbh I would suggest leaving the coordination part out until the rest works perfectly.

Reply:
if you want to keep it, definitely don't do the "undo at the end" thing.  just don't overwrite the original.